### ISSUE TITLE
openPMD-api writes wrong blocks in meshes - indexing bug

### ISSUE THREAD

**Issue by PrometheusPi** [ComputationalRadiationPhysics/picongpu#4641]
As discussed in https://github.com/openPMD/openPMD-api/issues/1485 PIConGPU seems to write incorrect blocks in openPMD causing shift in data and undefined regions. 

Please see the linked issue for details.

**Comment by PrometheusPi** [comment]
I have revised ticket 21796 at HZDR. Most likely the "empty blocks" were caused by the same problem. 
EDIT: While I see the same bug at iteration 0, later iteration are correct. Both problems seem to be independent.

**Comment by PrometheusPi** [comment]
The misalignment of written blocks only occurs at the 0th iteration. All other iterations cover the entire cell volume. 
Tested with:
```
for i in `seq 0 1000 10000`; do export FN=`printf simOutput_%06d.bp $i`; bpls -D $FN | grep fields/e_all_chargeDensity -A 33; done > test.txt
```
[test.txt](https://github.com/ComputationalRadiationPhysics/picongpu/files/12196520/test.txt)

@psychocoderHPC might this be caused by  `-m --windowMovePoint 0.0` ?

**Comment by psychocoderHPC** [comment]
Could you please provide you file `tbg/submit.start` I am wondering about your setup because I can not reproduce the isse.

**Comment by psychocoderHPC** [comment]
The error is not reproducible on our development system but on hemera. The issue is coming from an unset variable and therefore is random.
Fix is on the way.

bug introduced with #4585

**Comment by PrometheusPi** [comment]
This was fixed with #4644 - I will close the issue
